pyproject.toml

```diff
@@ -0,0 +1,116 @@
+# build-system
+# - ref: https://peps.python.org/pep-0517/
+#
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+
+# project
+# - ref 1: https://peps.python.org/pep-0621/
+# - ref 2: https://hatch.pypa.io/latest/config/metadata/#project-metadata
+#
+[project]
+name = "jupyterhub-pytest-plugin"
+description = "A reusable JupyterHub pytest plugin"
+readme = "README.md"
+requires-python = ">=3.5"
+license = {file = "LICENSE"}
+keywords = ["jupyterhub", "pytest", "plugin"]
+authors = [
+    {name = "Jupyter Contributors", email = "jupyter@googlegroups.com"},
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python :: 3",
+]
+# TODO: List the packages and pin them to a version that your project depends on
```

Comment:
I believe this, and also following `TODO` comments can be safely deleted.
Actually, instead of deleting them let's turn them into explanatory comments of the sections below.

Reply:
Done